invenio_app_rdm/theme/views.py

```diff
@@ -43,8 +43,10 @@ def create_blueprint(app):
         template_folder="templates",
         static_folder="static",
     )
-
-    blueprint.add_url_rule(**create_url_rule(routes["index"], default_view_func=index))
+    if app.config["RDM_FRONTPAGE"]:
```

Comment:
may i ask what the use case is to make the `/` route optional?
what will be rendered then as default if it is not configured?

Reply:
I would like to have custom view context, see
https://github.com/CERNDocumentServer/cds-rdm/pull/311/files#diff-1a67a7df3625c50413611bb5279e5be98b8b9b870aff748a0dde81062a0e3ceb

Reply:
For what it's worth, I've left a comment on the [CDS PR](https://github.com/CERNDocumentServer/cds-rdm/pull/313#discussion_r1914481728), suggesting a different approach for overriding views.

I can add this approach to docs, unless, we want to find a better approach so we're not limiting ourselves in the current 2-tuple format for the route values.